src/components/Header/Header.tsx

```diff
@@ -1,17 +1,43 @@
+import { useReactiveVar } from '@apollo/client';
+import MenuIcon from '@mui/icons-material/Menu';
+import { IconButton } from '@mui/material';
 import AppBar from '@mui/material/AppBar';
```

Comment:
one import is named and another is default

Reply:
fixed